alvr/session/src/settings.rs

```diff
@@ -1414,6 +1421,7 @@ This could happen on TCP. A IDR frame is requested in this case."#
     #[schema(gui(slider(min = 5, max = 1000, step = 5)), suffix = "ms")]
     pub minimum_idr_interval_ms: u64,
 
+    #[schema(strings(display_name = "DSCP"))]
```

Comment:
this could be a lot more descriptive, like `DSCP (packet prioritization hints)`, which still means nothing to the average person, but it's an improvement

Reply:
Made it a bit shorter, to avoid shifting the labels columns too far to the right